install-php-extensions

```diff
@@ -4545,6 +4545,17 @@ installRemoteModule() {
 			mv "$installRemoteModule_src" $(getPHPExtensionsDir)/tideways.so
 			installRemoteModule_manuallyInstalled=1
 			;;
+		translit)
+			if test -z "$installRemoteModule_version"; then
+				if test $PHP_MAJMIN_VERSION -lt 506; then
+					installRemoteModule_version=0.6.2
```

Comment:
You probably want to change this to `0.6.1` for `< 5.6`

Reply:
Thanks!